doc/k8s/sqlflow-mysql.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       volumes:
       containers:
-      - image: sqlflow/sqlflow
+      - image: sqlflow/sqlflow:latest
```

Comment:
Hi @MATRIX4284, I am curious why do we need this latest tag? I'd thought the latest is a tag used by default when there is no tag specified. In other words, `sqlflow/sqlflow` equals to `sqlflow/sqlflow:latest`. Am I wrong?

Reply:
Maybe we need to using the `digest` ID  to make sure the YAML always uses the same version on Kubernetes.
> Note: To make sure the container always uses the same version of the image, you can specify its digest, for example sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2. The digest uniquely identifies a specific version of the image, so it is never updated by Kubernetes unless you change the digest value.

> Note: You should avoid using the :latest tag when deploying containers in production as it is harder to track which version of the image is running and more difficult to roll back properly.

Ref: https://kubernetes.io/docs/concepts/configuration/overview/#container-images

@MATRIX4284 how about using a fixed digest ID to pin the Docker images version?